src/backend/cuda/CMakeLists.txt

```diff
@@ -101,11 +101,13 @@ cuda_include_directories(
   ${ArrayFire_BINARY_DIR}/include
   ${CMAKE_CURRENT_SOURCE_DIR}/kernel
   ${CMAKE_CURRENT_SOURCE_DIR}/jit
-  ${CMAKE_CURRENT_SOURCE_DIR}/cub
   ${ArrayFire_SOURCE_DIR}/src/api/c
   ${ArrayFire_SOURCE_DIR}/src/backend
   ${COMMON_INTERFACE_DIRS}
   )
+if(CUDA_VERSION_MAJOR VERSION_LESS 11)
+  cuda_include_directories(${ArrayFire_SOURCE_DIR}/extern/cub)
```

Comment:
Are they not updating the cub version on their GitHub repo? Can't we use the latest version on all older toolkits?

Reply:
They did, in fact we are quite behind in terms of those tags. CUB is at 1.9.10 and we are using 1.8.0. Testing it with CUDA 9.2, 10.2 and 11 now.